client/chat/create-channel.tsx

```diff
@@ -70,7 +70,9 @@ export function CreateChannel() {
     errorMessage = 'An error occurred while creating the channel.'
 
     if (isFetchError(error) && error.code) {
-      if (error.code === ChatServiceErrorCode.UserBanned) {
+      if (error.code === ChatServiceErrorCode.MaximumOwnedChannels) {
+        errorMessage = "You've reached the limit to how many channels you can create."
```

Comment:
Similar to the above, might be good to provide some remediation action here (e.g. suggest they leave or transfer ownership of a channel)